README.md

```diff
@@ -493,6 +493,12 @@ name,iterations,real_time,cpu_time,bytes_per_second,items_per_second,label
 "BM_SetInsert/1024/10",106365,17238.4,8421.53,4.74973e+06,1.18743e+06,
 ```
 
+## Output Files
+The library supports writing the output of the benchmark to a file specified
+by `--benchmark_output=<filename>`. The format of the output can be specified
+using `--benchmark_output_format={json|console|csv}`. Specifying
+`--benchmark_output` does not suppress the console output.
```

Comment:
why not just invoke and pipe to tee then? i'm not sure that this library should be writing files at all, given that every OS has a way of redirecting output to a file.